README.md

```diff
@@ -12,6 +12,14 @@ Please put all issues regarding IPFS _design_ in the
 [ipfs repo issues](https://github.com/ipfs/ipfs/issues).
 Please put all issues regarding go IPFS _implementation_ in [this repo](https://github.com/ipfs/go-ipfs/issues).
 
+## Security Issues
```

Comment:
There is a security note in https://github.com/ipfs/go-ipfs/blob/master/dev/crypto-notes.md, which I think should live in the issues in either ipfs/go-ipfs or ipfs/notes, instead of in the repo.